third_party/blink/renderer/modules/cobalt/h5vcc_accessibility/h_5_vcc_accessibility.cc

```diff
@@ -34,8 +35,11 @@ bool H5vccAccessibility::textToSpeech() {
 }
 
 // Called by browser to dispatch kTexttospeechchange event.
-void H5vccAccessibility::NotifyTextToSpeechChange() {
-  DispatchEvent(*Event::Create(event_type_names::kTexttospeechchange));
+void H5vccAccessibility::NotifyTextToSpeechChange(bool enabled) {
+  auto* event_init = TextToSpeechChangeEventInit::Create();
```

Comment:
I agree with the whole spirit of this PR (pending review), but 
wanted to bring @y4vor's comment/suggestion: could we cache
here the `enabled` value, e.g. in an 
`std:optional<boolean> text_to_speech_state_;` member variable,
and return that in `H5vccAccessibility::textToSpeech()` in l.29
if it's been initialized? That way we could prevent Kabuki from
initiating the blocking call (and thread hop) that wastes
performance cycles (because TTS state doesn't change too often).
WDYT?

Doing it in this PR or a subsequent one - I have no strong
preference.